sw/device/tests/entropy_src_fw_observe_many_contiguous.c

```diff
@@ -73,6 +73,20 @@ static status_t let_observe_fifo_overflow(uint32_t timeout_usec) {
   return OK_STATUS();
 }
 
+/**
+ * Determine whether health tested entropy bits have been dropped before the
+ * postht FIFO due to conditioner back pressure.
+ */
+bool postht_entropy_dropped(void) {
+  uint32_t recov_alert_sts;
+  CHECK_DIF_OK(
```

Comment:
NIT: Generally we prefer to `TRY` in functions and return a `status_t`, but I know this makes the API harder to use.

Reply:
You can probably leave it. There was a time when we wanted to move entirely over to `TRY` but we can do this in one larger effort later.